applications/MappingApplication/custom_utilities/mapping_matrix_utilities_mpi.cpp

```diff
@@ -83,14 +83,11 @@ void BuildMatrix(Kratos::unique_ptr<typename SparseSpaceType::MatrixType>& rpMdo
     EquationIdVectorType destination_ids;
     int ierr;
 
-    for (auto& rp_local_sys : rMapperLocalSystems)
-    {
+    for (auto& rp_local_sys : rMapperLocalSystems) {
```

Comment:
Could this loop be omp parallel? It would be good for hybrid parallelization, if we ever want that.

Reply:
I will add this in the future, it is also not there yet in the serial version since I want to test it properly

Also I haven't check the trilinos-doc on how to do it / if it is threadsafe